### PR TITLE
Added individual game stat viewing possibility & cleaned up individual league interface

### PR DESCRIPTION
**Pull request by bsankara**

### CODE REVIEW

File: public/js/league.js [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
   $("#btn-show-details").on("click", function() {
     window.location.href = window.location.pathname + "?detailed=true";
   });
   $("#btn-hide-details").on("click", function() {
     window.location.href = window.location.pathname + "?detailed=false";
   });
+
+
+  $('#individualGameStats').on("click", function() {
+    $(document.getElementsByClassName("table table-league")[0]).addClass('hide');
```
Comment (PeteyPii) [inline public/js/league.js]: Why not just use `$('table table-league').hide()`?

File: public/js/league.js [hunk 2]
```diff
@@ -1,8 +1,24 @@
...
+    $(document.getElementsByClassName("table table-league-individual hide")[0]).removeClass('hide');
+    $(this).addClass('hide');
+    $('#allGameStats').removeClass('hide');
+  });
+
+    $('#allGameStats').on("click", function() {
```
Comment (PeteyPii) [inline public/js/league.js]: Fix indentation

File: public/js/league.js [hunk 3]
```diff
@@ -1,8 +1,24 @@
 $(document).ready(function() {
+  var individualGames = false;
   $("#btn-show-details").on("click", function() {
     window.location.href = window.location.pathname + "?detailed=true";
   });
   $("#btn-hide-details").on("click", function() {
     window.location.href = window.location.pathname + "?detailed=false";
   });
```
Comment (PeteyPii) [inline public/js/league.js]: Mind removing this old stuff that isn't functional any more?

**Comment by PeteyPii** [comment]
Please use single quotation marks in js. Use jQuery .hide() and .show() rather than Bootstrap helper classes. I'd rather have styling in the CSS unless it's dynamic (so instead of starting off with class hide in the individual table, just make it a part of the CSS and then use jQuery .hide() and .show() ). jQuery does things more consistently and more nicely I think. I'm going to eventually get rid of the use of the text-alignment classes as well.

**Comment by PeteyPii** [comment]
Also, I think some 2 digit decimal precision would be nice for the per-game stats

**Comment by bsankara** [comment]
Made some fixes as you suggested

**Comment by PeteyPii** [comment]
Good now. Thanks for appeasing my demands.

**Pull request merged**